bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,237 @@
+import logging
+import random
+import typing
+from json import load
+from pathlib import Path
+
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import Colours, Lovefest
+
+log = logging.getLogger(__name__)
+
+HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+EMOJI_1 = random.choice(HEART_EMOJIS)
+EMOJI_2 = random.choice(HEART_EMOJIS)
+USER_LOVEFEST = []
+
+
+class BeMyValentine:
+    """
+    A cog that sends valentines to other users !
+    """
+    id = Lovefest.role_id
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="lovefest")
+    async def add_role(self, ctx):
+        """
+            This command adds people to the lovefest role
+        """
+        user = ctx.author
+        role = discord.utils.get(ctx.guild.roles, id=Lovefest.role_id)
+        if id not in [role.id for role in ctx.message.author.roles]:
+            USER_LOVEFEST.append(ctx.author)
+            await user.add_roles(role)
+            await ctx.send("The Lovefest role has been added !")
+        else:
+            await ctx.send("You already have the role !")
+
+    @commands.command(name='refreshlovefest')
+    async def refresh_user_lovefestlist(self, ctx):
+        """
+        Use this command to refresh the USER_VALENTINE list when the bot goes offline and then comes back online
+        """
+        USER_LOVEFEST.clear()
+        for member in ctx.guild.members:
+            for role in member.roles:
+                if role.id == Lovefest.role_id:
+                    USER_LOVEFEST.append(member)
+        embed = discord.Embed(
+            title="USER_LOVEFEST list updated!",
+            description=f'''The USER_LOVEFEST has been refreshed,`bemyvalentine` and `bemyvalentine dm` commands can now
+                            be used and there are {USER_LOVEFEST.__len__()} members having the lovefest role.''',
+            color=Colours.pink
+        )
+        await ctx.send(embed=embed)
+
+    @commands.cooldown(1, 1800, BucketType.user)
+    @commands.group(
```

Comment:
It's unnecessary for this to be spread across multiple lines; it's not an unreasonably long line if it's formatted as a single one.